grouper/fe/templates/permission-group.html

```diff
@@ -0,0 +1,176 @@
+{% extends "base.html" %}
+{% from 'macros/ui.html' import audit_log_panel, paginator, dropdown with context %}
+
+{% block subtitle %} permission {{ permission.name }}{% endblock %}
+
+{% block heading %}
+Permissions
+{% endblock %}
+
+{% block subheading %}
+{{ permission.name }} {{total}} permission(s)
+{% endblock %}
+
+{% block headingbuttons %}
```

Comment:
Not totally convinced that the grant-listing pages should have these buttons (or maybe even the audit log) -- seems easy to confuse the grants pages with the main permission page otherwise?